services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,18 +43,39 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
-
-  return ERR_CODE_SUCCESS;
```

Comment:
check to make sure that the queue has already been created

Reply:
just by checking that the pointer isn't null? like this?
  if (thermalMgrQueueHandle == NULL)
    return ERR_CODE_UNKNOWN;

Reply:
Yup